services/conference/src/scripts/stores/participants/LocalParticipant.ts

```diff
@@ -11,5 +11,11 @@ export class LocalParticipant extends ParticipantBase implements Store<ILocalPar
   @observable useStereoAudio = true
   constructor(id: string) {
     super(id)
+    const avatarSrc = localStorage.getItem('avatarSrc')
```

Comment:
The data seems should be extract from 'localParticipantInformation'

Reply:
I think init store with localStorage / urlParameters in same position would be better. So I've init local participant in `services\conference\src\scripts\models\middleware\initStores\localParticipant.ts`.